test/analysis_ee_ZH_cfg.py

```diff
@@ -273,6 +284,7 @@ def next():
     if simulator: 
         detector = simulator.detector
     if iev is not None:
+        pdebug.write(str('Event: {}\n'.format(iev)))
```

Comment:
put that in the debug analyzer.
the debug analyzer could be put at the beginning of the sequence.